Src/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -35,11 +35,15 @@ public class GoogleAuthorizationCodeFlow : AuthorizationCodeFlow
         /// <summary>Gets the token revocation URL.</summary>
         public string RevokeTokenUrl { get { return revokeTokenUrl; } }
 
+        /// <summary>Gets or sets the include granted scopes indicator.</summary>
+        private bool? includeGrantedScopes { get; set; }

```

Comment:
The reason I chose a nullable was to give the user a way to indicate if they wanted it to be true or false and also have it show up in the URI either way. The default behavior or a null indeed has the same end result of false, but if it was a standard bool and they for some reason wanted/needed the false to show up in the URI (validation checks on their end, maybe? grasping at straws, tbh) they would not be able to. I would not be opposed to having it be a standard bool since it doesn't affect the end result, I just tend to try and fit all scenarios just in case. Thoughts?